fe/fe-core/src/main/java/org/apache/doris/common/util/DebugUtil.java

```diff
@@ -135,6 +137,28 @@ public static String printId(final TUniqueId id) {
         return builder.toString();
     }
 
+    // id is a String generated by DebugUtil.printId(TUniqueId)
+    public static TUniqueId parseTUniqueIdFromString(String id) {
+        if (Strings.isNullOrEmpty(id)) {
+            throw new NumberFormatException("invalid query id");
+        }
+
+        String[] parts = id.split("-");
+        if (parts.length != 2) {
+            throw new NumberFormatException("invalid query id");
+        }
+
+        TUniqueId uniqueId = new TUniqueId();
+        try {
+            uniqueId.setHi(Long.parseUnsignedLong(parts[0], 16));
+            uniqueId.setLo(Long.parseUnsignedLong(parts[1], 16));
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException("invalid query id:" + e.getMessage());
```

Comment:
`throw new NumberFormatException("invalid id: " + id + " " + e.getMessage());`

Reply:
Caller should log more details, here just throw error.

Reply:
the id could be id of other task, not just query.